### DISCUSSION TITLE
Is there a postCSS plugin for the new @scope rule?

### DISCUSSION THREAD

**Discussion by SaroGFX** [postcss/postcss#1946] [Q&A]
I would imagine that the new [@scope](https://developer.mozilla.org/en-US/docs/Web/CSS/@scope#) rule can theoretically be polyfilled in the same way CSS modules is doing it. The only aspect is that it needs the correct reference from the source. 

Does anything like this exist, or is it being worked on?

**Comment by ai** [comment]
You should ask here (the CSS polyfills project):
https://github.com/csstools/postcss-plugins
